net/tstun/wrap.go

```diff
@@ -1173,6 +1182,9 @@ func (t *Wrapper) Write(buffs [][]byte, offset int) (int, error) {
 		if !t.disableFilter {
 			if t.filterPacketInboundFromWireGuard(p, captHook, pc) != filter.Accept {
 				metricPacketInDrop.Add(1)
+				metricInboundDroppedPacketsTotal.Add(dropPacketLabel{
+					Reason: DropReasonError,
```

Comment:
I think we don't need this now when we increment the same counter when `tdevWrite` returns a non-nil error.